docs/nidigital/rep_caps.rst

```diff
@@ -46,14 +46,14 @@ pins
         passes a string of :python:`'0, 1, 2'` to the set attribute function.
 
 
-devices
--------
+instruments
+-----------
 
-    .. py:attribute:: nidigital.Session.devices[]
+    .. py:attribute:: nidigital.Session.instruments[]
 
         .. code:: python
 
-            session.devices['0-2'].channel_enabled = True
+            session.instruments['0-2'].channel_enabled = True
```

Comment:
This is a terrible example, it will never work.
In config.py where we define the repeated capabilities container we should also define what to use for the example.
We should also use a real example not `channel_enabled` !
Shreyas please open an issue about this.

Reply:
https://github.com/ni/nimi-python/issues/1340